README.md

```diff
@@ -6,77 +6,77 @@ This release focus on the following:
+- We try to prepare for future features by introducing breaking changes now, allowing us to stay on v5 for as long as possible.
 
 # Major Changes
 
-## Removed deprecated things
+## Removed Deprecated Items
 
-All things that were deprecated in v4 were removed.
+All items deprecated in v4 were removed.
 
-MIGRATION: Make sure that your webpack 4 build don't print deprecation warnings.
+MIGRATION: Make sure that your webpack 4 build does not print deprecation warnings.
 
-Here are a few things that were removed but have no deprecation warning in v4:
+Here are a few things that were removed but did not have deprecation warnings in v4:
 
-- IgnorePlugin and BannerPlugin must be passed an options object
+- IgnorePlugin and BannerPlugin must now be passed an options object.
 
-## Automatic node.js polyfills removed
+## Automatic Node.js Polyfills Removed
 
-In the early days webpack aimed to allow to run most node.js modules in the browser, but the module landscape changed and many modules uses are now written for frontend purposes. webpack <= 4 ships with polyfills for many of the node.js core modules. They are automatically applied once a module uses i. e. the `crypto` module.
+In the early days, webpack's aim was to allow running most node.js modules in the browser, but the module landscape changed and many module uses are now written mainly for frontend purposes. webpack <= 4 ships with polyfills for many of the node.js core modules, which are automatically applied once a module uses any of the core modules (i.e. the `crypto` module).
 
-While this makes using modules written for node.js easy, it adds these huge polyfills to the bundle. In many cases these polyfills are even unneeded.
+While this makes using modules written for node.js easy, it adds these huge polyfills to the bundle. In many cases these polyfills are unnecessary.
 
-webpack 5 stopped to automatically polyfill these core modules and tried to focus on frontend-compatible modules.
+webpack 5 stops automatically polyfilling these core modules and focuses on frontend-compatible modules.
 
 MIGRATION:
 
-- Try to use frontend-compatible modules when possible.
-- It's possible to manually add the polyfill for a node.js core module. The error message gives a hint how to do that.
+- Try to use frontend-compatible modules whenever possible.
+- It's possible to manually add a polyfill for a node.js core module. An error message will give a hint on how to achieve that.
 - Package authors: Use the `browser` field in `package.json` to make a package frontend-compatible. Provide alternative implementations/dependencies for the browser.
 
-FEEDBACK: Please give feedback if you like or dislike this change. We are unsure if this will make it into the final release.
+FEEDBACK: Please provide us with feedback whether you like or dislike the above mentioned change. We are uncertain whether this will make it into the final release or not.
```

Comment:
@sokra we should also point to the place where users can give feedback by adding a link here.
Maybe an issue on this repo would work?

Reply:
I believe it is implicit.

Reply:
Having a single place for feedback is better for us as well as those giving feedback.

It's not that implicit, one can file issue on main webpack repo or on this repo or maybe tweet feedback :) .

Reply:
Actually there is already a place to give feedback which is here https://github.com/webpack/webpack/issues/8537 😅